README.md

```diff
@@ -31,6 +31,67 @@ Blade is the Design System that powers [Razorpay](https://razorpay.com/).
 - [Accessible](https://github.com/razorpay/blade/blob/master/rfcs/2022-04-09-accessibility.md#manual-testing)
 - Documented [RFCs](https://github.com/razorpay/blade/tree/docs/make-docs-pretty/rfcs) and [API Decisions](https://github.com/razorpay/blade/blob/master/packages/blade/src/components/Alert/_decisions/decisions.md)
 
+## Document global.navigator.product configuration for jest on React Native
```

Comment:
We should add this in installation guide instead of README - https://github.com/razorpay/blade/blob/master/packages/blade/docs/guides/Installation.stories.mdx

As people refer to installation guide mostly